docs/content/users/quickstart.md

```diff
@@ -211,6 +211,100 @@ ddev launch
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Kirby CMS
+
+Start a new [Kirby CMS](https://getkirby.com) project or use an existing one.
+
+=== "New projects"
+
+    Create a new Kirby CMS project from the official [Starterkit](https://github.com/getkirby/starterkit) using DDEV’s [`composer create` command](../users/usage/commands.md#composer):
+
+    ```bash
+    # Create a new project directory and navigate into it
+    mkdir my-kirby-project
+    cd my-kirby-project
+
+    # Set up the DDEV environment
+    ddev config --php-version=8.2 --omit-containers=db
+
+    # Spin up the project and install the Kirby Starterkit
+    ddev start
+    ddev composer create getkirby/starterkit
+
+    # Open the site in your browser
+    ddev launch
+    ```
+
+=== "Existing projects"
+
+    You can start using DDEV with an existing project as well:
+
+    ```bash
+    # Navigate to a existing project directory (or clone/download an existing project):
+    cd my-kirby-project
+
+    # Set up the DDEV environment
+    ddev config --php-version=8.2 --omit-containers=db
+
+    # Spin up the project
+    ddev start
+
+    # Open the site in your browser
+    ddev launch
+    ```
+
+=== "Public folder setup"
+
+    When possible, we recommend a [public folder setup](https://getkirby.com/docs/guide/configuration#custom-folder-setup__public-folder-setup) to move all files that need not be publicly accessible out of the doc root for enhanced security.
+
+    ```bash
+    # Clone a Starterkit into a new folder
+    git clone https://github.com/getkirby/starterkit my-kirby-project
+    
+    # Navigate into the new folder
+    cd my-kirby-project
+    
+    # Set up the DDEV environment with the docroot flag, this will create a public folder inside your project folder
+    ddev config --docroot=public create-docroot --php-version=8.2 --omit-containers=db
```

Comment:
```suggestion
    ddev config --docroot=public --php-version=8.2 --omit-containers=db
```
The current version of HEAD is getting the following error `Error: accepts 0 arg(s), received 1` executing `ddev config --docroot=public create-docroot --php-version=8.2 --omit-containers=db`.  While in the latest stable version of DDEV with v1.22.6 the `create-docroot` flag is still required (see https://github.com/ddev/ddev/pull/5768). I’ve quickly pinged @rfay and he agrees with my suggestion removing the `create-docroot`flag. he just said we have to be mindful and remember to pull it right at release time.

Reply:
@rpkoller Yep, when I tested yesterday, the command gave me an error when the folder didn't exist yet, so I added the `create-docroot` flag. Will remove it then.